pandas/core/generic.py

```diff
@@ -1116,16 +1102,25 @@ def f(x):
         else:
             return result.__finalize__(self)
 
-    def rename_axis(self, mapper, axis=0, copy=True, inplace=False):
+    def rename_axis(self, mapper=None, **kwargs):
         """
-        Alter the name of the index or columns.
+        Alter the name of the index or name of index backing the columns.
```

Comment:
I don't understand this change, is it a typo?

Oh, I think I saying "alter the name of the Index object that is the columns"?

If so, I think that `index` needs to be capitalized. But I think the older version was fine.

Reply:
I will use "alter the name of the Index object that is the columns."